scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -309,154 +309,230 @@ For this example we will focus on identifying DE genes in these `Tumor_Myoblast`
 
 ## Differential Expression Analysis
 
+Now we are ready to start preparing for our DE analysis, where we will compare the gene expression of tumor myoblast cells between ARMS and ERMS samples.
+
+Throughout the notebook we have been working with an integrated dataset that contains corrected gene expression data and a corrected UMAP.
+However, we do not want to use corrected gene expression values for differential expression; we want to use the original raw counts that are present in the `counts` assay of the `SingleCellExperiment` object. 
```

Comment:
We don't want `logcounts` for DESeq. It does not expect or use normalized data. We also aren't going to use logcounts for any kind of pseudobulking...

Reply:
This was not a clear comment from me and I meant to go back and change it, thanks for catching! I was trying to somehow communicate the difference between corrected and normalized since it felt odd to skip even talking about `logcounts` - including that we're not using it. Later I suggested to move a sentence up which is the better review!